reconcile/external_resources/manager.py

```diff
@@ -88,6 +88,56 @@ class ReconcileAction(StrEnum):
     DESTROY_ERROR = "Resource status in ERROR state"
 
 
+class ReconciliationStatus:
+    needs_secret_sync: bool = False
+    needs_deletion: bool = False
+    has_errors: bool = False
```

Comment:
`has_errors` could be a `@property` instead of manually setting it

Reply:
Added as a property